docs/basic/thresholding.md

```diff
@@ -0,0 +1,57 @@
+# Thresholding
+
+When flattening images and finding grains, TopoStats uses thresholding to separate the background data from the
+foreground data. This is done by setting a threshold value, and classifying all pixels above this value as foreground,
+and all pixels below this value as background.
+
+There are several different types of thresholding that can be used, and each has its own advantages and disadvantages.
+
+## Note: Thresholding above and below the surface
+
+TopoStats has the ability to threshold both above the sample surface and below it. This allows finding grains on the
+surface but also holes in the surface (useful for silicon wafer analysis). This can be configured by setting the
+corresponding "above" or "below" thresholds in the config file. Eg if you only want to find grains above the surface,
+only use the "above" threshold options, and vice-versa.
+
+## Thresholding types
+
+### Standard deviation thresholding
+
+Standard deviation thresholding is a simple method of thresholding that uses the standard deviation of the image to
+determine the threshold value. The threshold value is calculated as:
+
+$$
+\text{threshold} = \text{mean} + \text{std\_dev} \times \text{factor}
+$$
+
+Where `mean` is the mean of the image, `std_dev` is the standard deviation of the image, and `factor` is a user-defined
+value that determines how many standard deviations above the mean the threshold should be.
+
+This method is useful when you don't know the exact threshold value you want to use, and when you have a bit of noise
+in your image.
+
```

Comment:
This might be a great place for the histograms with a line and low opacity box to show the thresholded data

Reply:
Done - kinda - I haven't got data for each threshold to show the best cases for each threshold type for different types of images but I do have one visualisation